<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="champ_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>62</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="champ_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>192</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>540</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>champ_4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_calc">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>690</y>
     <width>360</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>champ_2+100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>770</y>
     <width>361</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
background-color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>137</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ-1 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>277</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ-2 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="champ_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>407</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Champ-3 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>640</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>label_5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="champ_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
